### PR TITLE
Add user validation

### PR DESCRIPTION
**Pull request by pento**
This is super hacky at the moment. 🙂

Run `composer install` to install the dependency. You may need to upgrade your local PHP version if you're using the default version in MacOS (`brew install php` will install PHP 8).

`wordpress-export.zip` is from the `data` directory. To change it, edit the files in the `data` directory, then run `cd data && rm ../wordpress-export.zip && zip -r ../wordpress-export.zip mimetype .`.

### CODE REVIEW

File: data/users/1.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (akirk) [inline data/users/1.json:1]: This is just test data for now, right? Should not be committed or put into a tests directory, right?

File: wxz-validator.php [hunk 2]
```diff
@@ -59,5 +69,24 @@ function raise_warning( $code, $message ) {
 				continue;
 			}
 		}
+
+		if ( 'users' === $type ) {
+			$result = $validator->validate( Helper::toJSON( $item ), 'https://wordpress.org/schema/user.json' );
```
Comment (akirk) [inline wxz-validator.php:80]: We can just use the file content, no?
```suggestion
			$result = $validator->validate( $file['content'], 'https://wordpress.org/schema/user.json' );
```

File: data/users/1.json (hunk 1 above)
Comment (pento) [reply data/users/1.json:1]: I just put it there while I was generating it: we should have more comprehensive data for testing, it's fine to just delete the `data` directory from this PR and set up better test data in a new PR.

File: wxz-validator.php (hunk 2 above)
Comment (pento) [reply wxz-validator.php:80]: Nope, per [the docs](https://opis.io/json-schema/2.x/php-validator.html#data-types), the data can't be a JSON-encoded string.

Comment (akirk) [reply wxz-validator.php:80]: Ah, right. But just using `$item` should work as per
```
// decoding a JSON string
$data = json_decode('{"name": "opis"}');
```

```suggestion
			$result = $validator->validate( $item, 'https://wordpress.org/schema/user.json' );
```

**Pull request merged**